### ISSUE TITLE
Implement "I had it that way!" for large replacables too

### ISSUE THREAD

**Issue by dgw** [zigdon/xkcd-Bucket#59]
When adding a duplicate value to a small replacable, Bucket will reply "$who, I had it that way!" and will not attempt to query the database to insert the dupe.

When adding a duplicate value for a large variable, when the value is not cached (at least that's when I've confirmed this), Bucket will simply say "Okay, $who" and then spew a DB Error regarding a duplicate entry.

**Comment by zigdon** [comment]
Yeah, I'd say either suppress the error (and fail silently) or check that we're updating a large var, and not say 'okay $who' until it succeeded.

I'd *avoid* saying 'okay $who', then later saying 'actually, I already had it that way.'
